R/init.R

```diff
@@ -531,27 +532,29 @@ if (interactive() &&
       environment()
     })
 
-    .First.sys <- function() {
-      # first load utils in order to overwrite its print method for help files
-      base::.First.sys()
+    if(!isFALSE(getOption("vsc.helpPanel", "Two"))){
```

Comment:
`isFALSE()` is added in R 3.5.0. Maybe we cannot rely on this function.

Reply:
Or we can conditionally import [backports](https://github.com/r-lib/backports). Certainly not a must have for `isFALSE`, but helps avoid such kind of issues.